docs/guide/advocacy.md

```diff
@@ -0,0 +1,89 @@
+This can be for various reasons, such as the project pulling the same dependencies in elsewhere.
+
+Showing a project dependency graph before and after can be more beneficial for this reason, and may help you find occurrences of target packages you weren't aware of.
+See also [Bundle Size](#bundle-size).
+
+### Speed
+
+If a cleanup is meant to improve the performance of a project, showing actual performance measurements for the project would be the best option followed by sharing a more general benchmark together with an explanation of why it is relevant such as the benchmarked method being called in a hot code path.
+Performance-oriented developers generally don't believe any comparisons not backed up with a reproducible experiment.
+
+Isolated benchmarks of a new dependency can be useful when the comparison is large (i.e. you're comparing against a _much_ slower dependency).
+However, in cases where the gap is small, it is often better to benchmark the project before and after instead of the dependency itself in isolation.
+
+Many maintainers just prefer a project benchmark too, so always refer to the contribution guide or ask the maintainers what their preference is.
+Downstream users often have very different usage scenarios and/or don't use some dependencies in hot paths.
+
+## What Isn't Convincing
+
+Most open source maintainers regularly deal with drive-by contributions that ignore their workflows and do a poor job of explaining themselves.
+Be careful not to make the common mistakes of those contributions, as that weakens the case for your change and for e18e as a whole.
+
+### Abstract Reports
+
+Don't be vague.
+Indicating that a cleanup could offer bug fixes, improved developer experience, or other features in the abstract is not convincing without any evidence those impact the project.
+
+* For bug fixes, be sure to have a reproduction -- much like any other kind of bug report
+* For developer experience and other features, you'll similarly want to be able to demonstrate why the new way is tangibly better
+
+### Downloads and Stars
+
+Non-user-facing metrics such as GitHub Stars and npm download statistics are not generally convincing.
+Both can easily be gamed and thus are not generally seen as reliable metrics.
+
+If your goal is to show that a package is well-known and trusted, it's better to demonstrate the factors that would result in those high numbers.
+Most often that is from showing lists of dependents that include many existing projects, especially other high-profile ones.
+
+### Support Ranges
+
+Some cleanups involve restricting project dependency ranges, such as raising a minimum Node.js version from an older version to the current LTS.
+Stricter dependency or platform support ranges on their own are not convincing, user-facing benefits.
+You'll want to instead demonstrate the tangible benefits that come from them.
```

Comment:
I'd go a bit further than saying that they're not a benefit on their own and say that they're a drawback
```suggestion
Stricter dependency or platform support ranges are a drawback to users as it may cause additional difficulty in upgrading the library.
You'll want to demonstrate that the tangible benefits that come from them are large enough for the majority of users that they they are worth the trade-off.
```